cobalt/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: cobalt_web_1
+      APP_PORT: 9001
+      PROXY_AUTH_ADD: "false"
+  web:
+    image: dennysubke/cobalt:10.6@sha256:870110e7002b7080adc81f9cee5357ab784adba81385a7b51663a7c1e92030cd
+    init: true
+    read_only: true
+    restart: on-failure
+    user: "1000:1000"
+    ports:
+      - 0.0.0.0:9000:9000
+    environment:
+      API_URL: "http://umbrel.local:9000/"
+      API_AUTH_REQUIRED: 0
+      DURATION_LIMIT: 86400
+    volumes:
+      - ${APP_DATA_DIR}/data/cookies:/cookies.json

```

Comment:
This would mount to a directory called `cookies.json' which would not work. From what I have seen in the documentation, this file would need to be pre-filled with tokens for various sites. I would leave this out for now.
```suggestion
```